.travis.yml

```diff
@@ -74,3 +74,8 @@ env:
   global:
     - secure: "pGQGM5YmHvOgaKihOyzb3k6bdqLQnZQ2OXO9QrfXlXwtop3zvZQi80Q+01l230x2psDWlwvqWTknAjAt1w463fYXPwpoSvKVCsLSSbjrf2l56nrDqnoir+n0CBy288+eIdaGEfzcxDiuULeKjlg08zrqjcjLjW0bDbBrlTXsb5U="
     - PIP_DISABLE_PIP_VERSION_CHECK=1
+    - CI_JIRA_URL="https://pycontribs.atlassian.net"
```

Comment:
Do not put credentails in plain text.

Reply:
Unfortunately this is a limitation of Travis. I've added all the variables to the [Travis settings](https://travis-ci.com/pycontribs/jira/settings) and dropped the env auth creds commit. This causes the build to fail on a `KeyError` as they are not passed to my fork for the PR.

This is by design from Travis CI:
https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions

If we expect external contributors to fork the project and have their PRs run these same tests, we necessarily must expose the credentials to these users.

Reply:
One option would be to obfuscate them, e.g. store them encrypted together with the password. Security through obfuscation doesn't make it safer, but at least it requires someone to put a bit of effort in to get them and makes it harder for bots to scrape them.

Reply:
@ssbarnea Would Addono's proposal be an acceptable solution? I'm not certain there's any real workaround to this as it's a hard limitation of Travis. If having the credentials to the cloud test instance in the code (even obfuscated) are completely unacceptable then our efforts might be better redirected towards provisioning local test instances as part of the CI.

Reply:
@ssbarnea Hi, just bumping this into view since it looks like this has been sitting for a bit and it appears to be crucial for this PR.